[RJA-1535][fix]: amazon ssh port check - catch Authorization exception

ddurnev opened the pull request.

Commit:
[RJA-1535][fix]: amazon ssh check - catch Authorization exception

bugzmanov commented on commons/src/main/scala/com/griddynamics/executors/provision/CommonSshPortChecker.scala:
```diff
@@ -40,20 +41,23 @@ trait CommonSshPortChecker extends AsyncTimeoutAwareActionExecutor with Logging
+        None
 
-      }
+      case ae : AuthorizationException =>
```
I think it would be usefull to log this kind of exceptions either to step or action log

Commit:
[RJA-1585][fix]: ssh port checker action fix
Reduce default timeout to 20 sec(added separate plugin property)
Log "auth fail" exceptions to action log
